### DISCUSSION TITLE
about MusicFontSymbol.ts

### DISCUSSION THREAD

**Discussion by brianpenghe** [CoderLine/alphaTab#670] [Q&A] [answered]
I was trying to understand the coding of different font symbols.
In the [MusicFontSymbol.ts](https://github.com/CoderLine/alphaTab/blob/develop/src/model/MusicFontSymbol.ts) file, it seems that the 6-digit string represents individual symbols such as `TimeSig0 = 0xe080`. However, in [this pdf file](https://www.unicode.org/charts/PDF/U1D100.pdf) I see a 5-digit string. 
I assume that they two should be part of the same system. May I ask how are they inter-translated?

Thanks

**Comment by Danielku15** [comment, marked as answer]
As the comment on the enum indicates: 
> Lists all music font symbols used within alphaTab. **The names and values are aligned with the SMuFL standard**. 

There is no direct relation between the [SMuFL standard](https://www.smufl.org/) and the general Unicode range for music symbols. I guess it would be possible to create some sort of mapping for some of the symbols which exists in both standards.
